<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesetLightEntryWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="entryHorizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="LineEditWidget" name="firstSubtileLineEdit">
       <property name="charWidth">
        <number>5</number>
       </property>
       <property name="toolTip">
        <string>Index of the first subtile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEditWidget" name="lastSubtileLineEdit">
       <property name="charWidth">
        <number>5</number>
       </property>
       <property name="toolTip">
        <string>Index of the last subtile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEditWidget" name="lightLineEdit">
       <property name="charWidth">
        <number>3</number>
       </property>
       <property name="toolTip">
        <string>Light radius of the selected subtiles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
